staging/src/k8s.io/client-go/tools/reference/ref.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package ref
+package reference
```

Comment:
In my mind tools will be for features that deserve a README package and an example in /examples. It the reality is move further away from this picture.